dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ProcessRestRepository.java

```diff
@@ -20,6 +20,7 @@
 import org.dspace.core.Context;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.domain.Pageable;
+import org.springframework.data.rest.webmvc.ResourceNotFoundException;
 import org.springframework.stereotype.Component;
 
 /**
```

Comment:
/**
     * This method will return a list of all Process objects converted to ProcessRest objects
     * @param pageable
     * @return  The list of ProcessRest objects coming forth from all Process objects in the database
     * @throws SQLException If something goes wrong
     */